Constant.py

```diff
@@ -0,0 +1,52 @@
+BATTERY_PARAM_CHARACTERISTICS = {
+    'cell_temperature_in_celsius': {
```

Comment:
Great data abstraction. Separation of individual items is also good :)

Reply:
Thanks :)